### PR TITLE
Fix Layout/IndentHeredoc, Consistency, IndentationWidth, and new violations

### PR DESCRIPTION
**Pull request by jaysonesmith**
## Details

Fixed
- Layout/IndentHeredoc
- Layout/IndentConsistency
- Layout/IndentIndentationWidth
- New violations from above fixes

## Motivation and Context

Working to help solve issue [1021](https://github.com/cucumber/cucumber-ruby/issues/1021)!

## How Has This Been Tested?

`bundle exec rake` :+1:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Rubocop style fixes

### CODE REVIEW

File: lib/cucumber/formatter/html.rb [hunk 1]
```diff
@@ -590,7 +590,7 @@ def lines_around(file, line)
           rescue ArgumentError
             return "# Couldn't get snippet for #{file}"
           end
-          min = [0, line-3].max
+            min = [0, line-3].max
```
Comment (luke-hill) [inline lib/cucumber/formatter/html.rb]: I'm not convinced on the tabbing above for this file. It's hella complicated.

589-592 I think need one tab

File: spec/cucumber/multiline_argument/data_table_spec.rb [hunk 2]
```diff
@@ -254,10 +254,10 @@ module MultilineArgument
 
       describe '#map_headers' do
         let(:table) do
-           DataTable.from([
-          %w{HELLO WORLD},
-          %w{4444 55555}
-          ])
+          DataTable.from([
```
Comment (luke-hill) [inline spec/cucumber/multiline_argument/data_table_spec.rb]: these feel off? Think this change needs undoing (1 space here?)

Comment (jaysonesmith) [reply spec/cucumber/multiline_argument/data_table_spec.rb]: Yeah, there's a couple that somehow ended up funky.

File: lib/cucumber/formatter/html.rb (hunk 1 above)
Comment (jaysonesmith) [reply lib/cucumber/formatter/html.rb]: Yeah, looks like a phantom tab or something. Will fix. Thanks for the lookout!

Comment (jaysonesmith) [reply lib/cucumber/formatter/html.rb]: Fixed in 039b5f1 :)

**Comment by jaysonesmith** [comment]
@olleolleolle Could I get a 👍 on this one too, please?

**Comment by jaysonesmith** [comment]
Tyvm!

**Pull request merged**